src/coreclr/inc/regdisp.h

```diff
@@ -135,23 +135,30 @@ inline LPVOID GetRegdisplayFPAddress(REGDISPLAY *display) {
     return (LPVOID)display->GetEbpLocation();
 }
 
+inline TADDR GetRegdisplayPCTAddr(REGDISPLAY *display)
+{
+    return display->PCTAddr;
+}
+
 inline void SetRegdisplayPCTAddr(REGDISPLAY *display, TADDR addr)
```

Comment:
I initially tried to optimize how the `PCTAddr` is stored for callee context to avoid recomputation. Unfortunately it results in some bugs (eg. `CopyRegDisplay` doesn't transfer the information correctly, `IsInCalleesFrames` depends on quirks in the initial state, etc.) and I opted not to pursue this optimization at this time. I left the helper methods in place to make it easier to address this in future and to create a central place where we can `_ASSERTE` on invalid values.